depthai_nodes/ml/messages/segmentation.py

```diff
@@ -5,26 +5,25 @@
 
 class SegmentationMask(dai.Buffer):
     """SegmentationMask class for a single- or multi-object segmentation mask.
-    Background is represented with "-1" and foreground classes with non-negative
-    integers.
+    Background is represented with "0" and foreground classes with positive integers.
```

Comment:
Let's keep actual object classes as they were, starting with 0 and let's make background be the maximum datatype value. So if mask: NDArray[np.uin16] then background is `65535`. You can get this value with:
`np.iinfo(np.uint16).max.`
This way values in the mask correspond 1:1 with instance detections and we also don't use negative number which is more device efficient. Thoughts?

Reply:
Is there a reason we dont go with 0 as the background class?

Reply:
Yeah, main reason why we don't want this is the case for instance segmentation. There we have list of detections and a joint mask for all of them. And ideally we want the object at index 0 have it's instance mask encoded with index 0.

Reply:
Okay, thanks for clarification!

Reply:
Added in [8b6db71](https://github.com/luxonis/depthai-nodes/pull/127/commits/8b6db71f66f0dae8cde8be2f9527f90501711da8).